bin/bob

```diff
@@ -40,6 +40,86 @@ multi sub MAIN("dist", *@paths, Bool :$force = False)
 	}
 }
 
+multi sub MAIN("new", Str $module-name = "", Bool :$git = True)
+{
+	my Str $name = $module-name;
+
+	# Ask for a module name if one was not supplied yet
+	if ($name eq "") {
+		$name = ask("Name of the module");
+	}
+
+	# Create a directory name for the module
+	my $dir-name = "perl6-" ~ $name.subst("::", "-", :g);
+
+	# Make sure it isn't already taken on the local system
+	if ($dir-name.IO.e) {
+		die "Directory named $dir-name already exists.";
+	}
+
+	# Ask the user about some information on the module
+	my $author = ask("Your name");
+	my $email = ask("Your email address");
+	my $perl = ask("Perl 6 version", :default("c"));
+	my $description = ask("Module description", :default("Nondescript"));
+	my $license = ask("License key", :default("GPL-3.0"));
+
+	# Create the initial %meta
+	my %meta = %(
+		meta-version => 1,
+		perl => "6.$perl",
+		name => $name,
+		description => $description,
+		authors => ("$author <$email>"),
+		tags => (),
+		version => "0.0.0",
+		provides => (),
+		depends => (),
+	);
+
+	# Create the module skeleton
+	mkdir $dir-name;
+	chdir $dir-name;
+	mkdir "bin";
+	mkdir "lib";
+	mkdir "t";
+
+	my $gitignore = q:to/EOF/
```

Comment:
@samcv Would you have any ideas to neatly turn this (and other) templates into their own files? Seems like clutter to have them in the source code directly.